### PR TITLE
Oladimeji file upload

### PR DESCRIPTION
**Pull request by ojokure**
##  What does this PR do

As a user I want to be able to change/Upload my profile picture

##  Description of task to be completed

profile picture update 

##  How should this manually be tested

go to settings and initiate a profile pic change

##  Questions (If any):

### CODE REVIEW

File: src/state/actions/authenticationActions.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ export const login = (props, values) => dispatch => {
   axios
     .post(`${url}user/login`, values)
     .then(res => {
+      console.log(res);
```
Comment (BenjaminGrabow) [inline src/state/actions/authenticationActions.js]: Let's delete all console.logs from our PR.

File: src/state/actions/settingActions.js [hunk 2]
```diff
@@ -28,3 +28,34 @@ export const updateUserInfo = (
...
+//     )
+//     .then(res => {
+//       setUserInfo({ ...userInfo, avatar_url: res.data.secure_url });
+//       console.log('5', res.data.secure_url);
+//       console.log('7', userInfo);
+//     })
+//     .catch(err => {
+//       console.log(err);
+//     });
+// };
```
Comment (BenjaminGrabow) [inline src/state/actions/settingActions.js]: Let's delete all comments from our PR.

File: src/views/Settings/Settings.js [hunk 3]
```diff
@@ -89,9 +107,68 @@ function Settings(props) {
...
+      )
+      .then(res => {
+        const secureUrl = res.data.secure_url;
+        setUserInfo({
+          ...userInfo,
+          avatar_url: secureUrl,
+        });
+      })
+      .catch(err => {
+        console.log(err);
```
Comment (BenjaminGrabow) [inline src/views/Settings/Settings.js]: Let's delete all console.log from our PR.

File: src/state/actions/settingActions.js (hunk 2 above)
Comment (ojokure) [reply src/state/actions/settingActions.js]: deleted

File: src/views/Settings/Settings.js (hunk 3 above)
Comment (ojokure) [reply src/views/Settings/Settings.js]: cleared

File: src/state/actions/authenticationActions.js (hunk 1 above)
Comment (ojokure) [reply src/state/actions/authenticationActions.js]: cleared

**Comment by BenjaminGrabow** [comment]
[As a user, I want to be able to upload a picture from me](https://trello.com/c/caA4UZRR/107-as-a-user-i-want-to-be-able-to-upload-a-picture-from-me)

**Pull request merged**